definitions/divorce/json/CaseTypeTab/CaseTypeTab.json

```diff
@@ -1456,7 +1456,7 @@
     "TabLabel": "AOS",
     "TabDisplayOrder": 3,
     "CaseFieldID": "ReceivedAOSfromResp",
-    "TabFieldDisplayOrder": 22
+    "TabFieldDisplayOrder": 26
```

Comment:
are you sure you want to update prod?

Reply:
@yarpo need to display some new nonprod fields before the existing fields, the alternative would be to move these to a prod file and duplicate in a nonprod file which I'm happy to do but the behaviour will be the same